packages/pebble-shared/svgs/checked-server.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="1000" height="1000" viewBox="0 0 1000 1000" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
- size should be 1000x1000
- color should be black (or no color)

<img width="494" alt="Screenshot 2022-06-22 at 11 51 50 AM" src="https://user-images.githubusercontent.com/8132723/174957673-1b4a6bd8-10cf-4f33-b071-3e99b9182b78.png">